### PR TITLE
Add syslog-tag option

### PR DESCRIPTION
**Pull request by LK4D4**
Also I fixed some docs glitches. Ping @moxiegirl for this.
Close #12668

### CODE REVIEW

**Comment by calavera** [comment]
I like the simplicity of this! LGTM

**Comment by runcom** [comment]
LGTM

**Comment by LK4D4** [comment]
@runcom Code is frozen!

**Comment by runcom** [comment]
@LK4D4 I missed that :cry: :sob:

**Comment by moxiegirl** [comment]
LGTM

I did spot some things but they existed before you hit the file.  Ping me back if you want to fix some stuff otherwise I'll kill the minor issues another, less long, day.

**Comment by moxiegirl** [comment]
ping @docker/docs-owners

**Comment by thaJeztah** [comment]
LGTM

@LK4D4 is this a backward-incompatible change to the `address` option? Or is that not in the current release?

**Comment by LK4D4** [comment]
@thaJeztah There was no `address` option I think. At least I hope :)

File: docs/sources/reference/run.md [hunk 1]
```diff
@@ -882,20 +882,27 @@ command is not available for this logging driver
...
 The following logging options are supported for this logging driver:
 
-    --log-opt address=[tcp|udp]://host:port
-    --log-opt address=unix://path
+    --log-opt syslog-address=[tcp|udp]://host:port
+    --log-opt syslog-address=unix://path
+    --log-opt syslog-tag="mailer"
 
-`address` specifies the remote syslog server address where the driver connects to.
+`syslog-address` specifies the remote syslog server address where the driver connects to.
```
Comment (thaJeztah) [inline docs/sources/reference/run.md]: @LK4D4 this change was what made me think it was there before (but I may be confused)

Comment (LK4D4) [inline docs/sources/reference/run.md]: I thought it is docs bug. ping @runcom

Comment (runcom) [inline docs/sources/reference/run.md]: @LK4D4 yup there's `syslog-address` (my fault) https://github.com/docker/docker/blob/master/daemon/logger/syslog/syslog.go#L37

Comment (thaJeztah) [inline docs/sources/reference/run.md]: thanks @runcom! no problem then :)

**Comment by psquickitjayant** [comment]
Is there any documentation where we can set the custom tag for syslog or it just puts the container id as syslog tag.

**Comment by thaJeztah** [comment]
@psquickitjayant you can find the relevant changes in this PR: 
https://github.com/docker/docker/pull/13672/files. The docs changes haven't been published yet, because this feature is not yet available in the current (1.6.2) release. This feature will be part of the 1.8 release (or the 1.7 "experimental" builds)

**Comment by psquickitjayant** [comment]
thanks @thaJeztah

**Comment by psquickitjayant** [comment]
@thaJeztah  does it also support syslog or just syslog-ng?

**Comment by mostlyjason** [comment]
@psquickitjayant I think you mean rsyslog?

**Comment by psquickitjayant** [comment]
thanks @mostlyjason  I have updated my comment.

**Comment by runcom** [comment]
It supports rsyslog as well

**Comment by jawher** [comment]
Any chance of this getting included in one of the point release now that 1.7 is out (instead of waiting for 1.8) ?

**Comment by ekristen** [comment]
I second @jawher, any chance this can make it into a point release!

**Comment by thaJeztah** [comment]
@ekristen really sorry, but the dot-releases are only for bugfixes. you can try the experimental builds though

**Pull request merged**